docs/book/rest-api.md

```diff
@@ -1648,7 +1648,71 @@ The effective path of the JSON Patch operation is obtained by joining the path p
 
 ## <a name="query-api"></a> Query API
 
-### <a name="execute-a-query"/>Execute a Query
+### <a name="execute-a-simple-query"/>Execute a Simple Query
+
+```
+POST /
+Content-Type: application/json
+```
+
+Execute a simple query.
+
+Callers can execute POST requests against the server without any path in the
+URL. If the server has policies that define a document at path
+`/data/system/main`, that document will be returned.
```

Comment:
It should be under data. I'll add a clarifying comment like we have for the authorization policy that lives under `system.authz`.

I could imagine starting a new docs page to cover OPA administration topics. The system namespace would be explained in more detail there. Once we have a few more of these, it will make sense to create that. Some that jump to mind are metrics, tracing, logging, profiling, etc.